rt/rest2.py

```diff
@@ -748,6 +754,26 @@ def reply(self,
 
         return bool(msg[0])
 
+    def delete_ticket(self, ticket_id: typing.Union[str, int]) -> None:
+        """ Mark a ticket as deleted.
+
+        :param ticket_id: ID of ticket
+
+        :raises BadRequestError: When user does not exist
+        :raises NotFoundError: If the user does not exist
+        :raises UnexpectedResponseError: If the response from RT is not as expected
+        """
+        try:
+            _ = self.__request_delete(f'ticket/{ticket_id}')
```

Comment:
Why storing the return value if it's not used?

Reply:
absolutely, fixed